Source/Core/Core/GeckoCode.cpp

```diff
@@ -275,10 +276,10 @@ void RunCodeHandler()
     PowerPC::HostWrite_U64(rPS(i).PS0AsU64(), SP + 24 + 2 * i * sizeof(u64));
     PowerPC::HostWrite_U64(rPS(i).PS1AsU64(), SP + 24 + (2 * i + 1) * sizeof(u64));
   }
-  DEBUG_LOG(ACTIONREPLAY,
-            "GeckoCodes: Initiating phantom branch-and-link. "
-            "PC = 0x%08X, SP = 0x%08X, SFP = 0x%08X",
-            PC, SP, SFP);
+  DEBUG_LOG_FMT(ACTIONREPLAY,
+                "GeckoCodes: Initiating phantom branch-and-link. "
+                "PC = {:#010X}, SP = {:#010X}, SFP = {:#010X}",
```

Comment:
Won't these changes produce `0XHHHHHHHH` (uppercase `X`) rather than the initial `0xHHHHHHHH` (lowercase `x`).